ydb/library/yql/core/common_opt/yql_co_pgselect.cpp

```diff
@@ -924,6 +924,19 @@ TUsedColumns GatherUsedColumns(const TExprNode::TPtr& result, const TExprNode::T
         for (ui32 i = 0; i < groupTuple->ChildrenSize(); ++i) {
             auto join = groupTuple->Child(i);
             auto joinType = join->Child(0)->Content();
+            if (join->ChildrenSize() > 1 && join->Child(1)->Content() == "using") {
```

Comment:
please validate IsAtom precondition && atom's content & here too (only "using" is supported now)